<launch>
    <node pkg="task_master" type="task_ctrl" name="task_ctrl_node" output="screen"/>
    <node pkg="task_master" type="wp_filter_sender" name="wp_filter_sender" output="screen"/>
    <node pkg="task_master" type="mavros_2_gazebo_conversion" name="mavros_2_gazebo_conversion" output="screen"/>
</launch>

